ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -25,12 +25,12 @@
 
 - name: Set homebrew path (x64)
   set_fact:
-    homebrew_path: /usr/local/bin/brew
+    homebrew_path: /usr/local/bin
```

Comment:
will this leave old installs lying about?

Reply:
@karianna I believe we're ok. `homebrew_path` is being changed from the executable location to the directory that the executable was in. The only two previous references to this variable were the check for its presence and one invocation, which have been modified. All the others are new references and I believe the `/usr/local/bin` and `/opt/homebrew/bin` locations are the defaults so this should not conflict with existing installation. I'll be honest the use of the `/bin` in these paths looks a bit strange to me, but that is consistent with the [ansible docs](https://docs.ansible.com/ansible/latest/collections/community/general/homebrew_module.html#parameter-path).